app/javascript/dashboard/components/widgets/conversation/TagAgents.vue

```diff
@@ -13,41 +14,103 @@ const props = defineProps({
 
 const emit = defineEmits(['selectAgent']);
 
+const { t } = useI18n();
 const getters = useStoreGetters();
 const agents = computed(() => getters['agents/getVerifiedAgents'].value);
+const teams = computed(() => getters['teams/getTeams'].value);
```

Comment:
Let us use `useMapGetter` for both agents and teams
```suggestion
const teams = useMapGetter('teams/getTeams');
```